test/views/ChoiceViewSpec.scala

```diff
@@ -181,28 +181,28 @@ class ChoiceViewSpec extends AppViewSpec with ChoiceMessages with CommonMessages
   }
   private def ensureAllLabelTextIsCorrect(view: Html): Unit = {
     val labels = view.getElementsByTag("label").toList
-    labels.forall(elems => elems.getElementsContainingText(messages(supplementaryDec)).isEmpty) must be(false)
-    labels.forall(elems => elems.getElementsContainingText(messages(standardDec)).isEmpty) must be(false)
-    labels.forall(elems => elems.getElementsContainingText(messages(cancelDec)).isEmpty) must be(false)
-    labels.forall(elems => elems.getElementsContainingText(messages(recentDec)).isEmpty) must be(false)
-    labels.forall(elems => elems.getElementsContainingText(messages(continueDec)).isEmpty) must be(false)
+    labels.forall(elems => elems.getElementsContainingText(messages(supplementaryDec)).isEmpty) mustBe false
```

Comment:
every(elems.getElementsContainingText(messages(supplementaryDec)) mustBe empty

Reply:
Error message would be terrible with `forall` on collection - like `true is not false`
There is [some scalatest combinator/methods](http://www.scalatest.org/user_guide/using_matchers#inspectorShorthands) which would provide nice errors for collections

Reply:
This has been addressed in a simpler way